<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-T1H5H3C" failures="1" tests="1" name="runner.Runner" time="6.305" errors="0" timestamp="2024-04-16T17:18:16 IST" skipped="0">
  <testcase classname="runner.Runner" name="runScenario" time="6.305">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at stepDefinitions.Login.dashboard_screen_is_displayed(Login.java:48)
at âœ½.Dashboard screen is displayed(file:///C:/Users/Admin/eclipse-workspace/zframeworkProject/src/test/java/features/Login.feature:29)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runner.Runner -->
